.githooks/components.sh

```diff
@@ -9,5 +9,6 @@ components=( \
 	["infra"]="Makefile",".githooks/","hack/","openshift-ci/" \
 	["vendor"]="vendor","cnf-tests","ztp","go.mod","go.sum" \
 	["cnf-tests"]="cnf-tests"
+	["owners"]="OWNERS","OWNERS_ALIASES"
```

Comment:
yeah,
this was overlooked in the initial components design.
I agree owners deserve it's own component.

just make sure to add ' \' to the line above for consistency